sklearn/model_selection/_validation.py

```diff
@@ -464,6 +464,12 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
         score_time = 0.0
         if error_score == 'raise':
             raise
+
+        elif error_score == 'raise-deprecating':
+            warnings.warn("Estimator fit failed. From version 0.22 "
+                          "the default value will be error_score=np.nan",
```

Comment:
I suggested a more verbose error message here because I don't believe that every user who encounters this message will be familiar with error_score or appreciate what "the default value will be" means to them